libs/platform/src/lib/components/input-group/input-group.component.ts

```diff
@@ -0,0 +1,159 @@
+import {
```

Comment:
Input group is not a input component? or it is just a wrapper? Why not extending baseInput and adding this component under forms folder ?

Reply:
There is no requirement that it should a form component itself in the [spec](https://fundamental-ngx.gitbook.io/docs/v/master/input-group-2)
 
Following the spec tech it's considered just as a wrapper to add some extra context to an user. 
`fdp-input` should be put inside with all its form bindings.

Probably I need to add example to show usage withing form

Reply:
@fkolar In this case I'm going to implement it as a standalone form component.

Reply:
redone it, now it implements `BaseInput` interface